### PR TITLE
[search] Fixed streets matching by postcodes.

### PR DESCRIPTION
**Pull request by ygorshenin**
https://jira.mail.ru/browse/MAPSME-1630

### CODE REVIEW

File: search/geocoder.cpp [hunk 1]
```diff
@@ -1213,7 +1213,23 @@ void Geocoder::MatchPOIsAndBuildings(size_t curToken)
 
     // If there're only one street layer but user also entered a
     // postcode, we need to emit all features matching to postcode on
-    // the given street.
+    // the given street, including the street.
```
Comment (mpimenov) [inline search/geocoder.cpp]: including the street itselft.

Comment (ygorshenin) [inline search/geocoder.cpp]: Done

**Comment by ygorshenin** [comment]
PTAL

**Comment by mpimenov** [comment]
LGTM

**Comment by ygorshenin** [comment]
Thanks!

**Pull request merged**